### ISSUE TITLE
Gitlab token exposed in _config

### ISSUE THREAD

**Issue by alkuzad** [JustinAiken/jekyll-gitlab-letsencrypt#4]
Do you think providing options for fetching it directly from ENV is good option ? There is no easy way to use ENV variables in _config.yml.

I've did this locally and code is pushed but token is not exposed.

**Comment by JustinAiken** [comment]
- Could always read it like this:
```yaml
gitlab-letsencrypt:
  gitlab_repo: "foo/bar"
  personal_access_token: <%= ENV['GITLAB_TOKEN'] %>
```
- Or just `.gitignore` `_config.yml`, and have a `_config.example.yml` checked into source control..
- Or do you mean make the plugin itself look to the ENV, so you'd run it as `GITLAB_TOKEN=muchsecret jekyll letsencrypt` ?

**Comment by alkuzad** [comment]
method is not crucial, I just want to pass it in method that won't expose token. _config file is gitlab controled file. As far as I know _config.yml is not pre-parsed and you can't use erb markup in there. Or is it possible ?

**Comment by JustinAiken** [comment]
@alkuzad - See if #6 looks good to you.  I just tried it out renewing the cert for my domain, it seemed to work.

**Comment by JustinAiken** [comment]
Released as [0.1.0](https://github.com/JustinAiken/jekyll-gitlab-letsencrypt/releases/tag/v0.1.0)
